indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1439,112 +1440,65 @@ def firstPage(self, c, doc):
     def getBody(self, story=None, indexedFlowable={}, level=1 ):
```

Comment:
I think we always pass a value for indexedFlowable. Can you replace the default value with `None` to avoid a possible change to a mutable default arg?